canonical/assets.ubuntu.com
# upgrade vanilla framework

## Done

- Upgrade NPM packages
- Upgrade to Vanilla V3

## QA

- Check out this feature branch
- Run the site using the command `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8017
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Make sure that the search view and the assets creation view aren't broken visually

## Screenshots

![0 0 0 0_8017_manager_create](https://user-images.githubusercontent.com/36013798/231187505-4050e316-32ca-4278-b8d5-b074ae627a23.png)
